delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/Event.scala

```diff
@@ -23,6 +24,11 @@ trait Event extends Product with Serializable {
     * @return the subject that performed the action that resulted in emitting this event
     */
   def subject: Subject
+
+  /**
+    * @return Some(project) when an event belongs to a project, None otherwise
+    */
+  def belongsTo: Option[ProjectRef]
```

Comment:
Maybe we could create a subtype of events like `ProjectBasedEvent` with just `def project: ProjectRef` rather than add this field to all the events, even those that don't belong to a project.

Reply:
I'd prefer the method to the sub-type, honestly...

Reply:
I'm fine with this, but I didn't find a good name for it. `ProjectBasedEvent` might be alright...but it is not very convincing either.

Reply:
how about `ProjectScopedEvent`?